src/database.ts

```diff
@@ -448,6 +436,14 @@ class Database extends ServiceObject {
    *   const apiResponse = data[1];
    * });
    */
+  createSession(
```

Comment:
looks like they use consistent template =>
overloads => documentation and examples => implementation.
We shouldn't move these things.

Reply:
I have review some of files about what standard they use:
in `database.ts`, `session.ts` and `table.ts` they use
**documentation and examples => overloads => implementation**
see below some examples
[database.ts](https://github.com/googleapis/nodejs-spanner/blob/3aa676d7bbc8e846da8b8ff518ad5998f03ff933/src/database.ts#L544)
[session.ts](https://github.com/googleapis/nodejs-spanner/blob/3aa676d7bbc8e846da8b8ff518ad5998f03ff933/src/session.ts#L268)
[table.ts](https://github.com/googleapis/nodejs-spanner/blob/3aa676d7bbc8e846da8b8ff518ad5998f03ff933/src/table.ts#L147)

only in `transaction.ts` they use 
**overloads => documentation and examples => implementation**
[transaction.ts](https://github.com/googleapis/nodejs-spanner/blob/3aa676d7bbc8e846da8b8ff518ad5998f03ff933/src/transaction.ts#L234)

Currently, putting it like **documentation and examples => overloads => implementation**
**Suggestion required**